moped-editor/src/views/projects/projectView/ProjectComponentsMap.css

```diff
@@ -18,16 +18,6 @@
   height: 4rem !important;
 }
 
-.MuiFab-sizeSmall:nth-of-type(1) {
-  background-image: url(/moped/static/images/mapStreets.jpg) !important;
-  color: black;
-}
-
-.MuiFab-sizeSmall:nth-of-type(2) {
-  background-image: url(/moped/static/images/mapAerial.jpg) !important;
-  color: white;
-}
-
```

Comment:
TLDR, it's now impossible to reference files in `/public` directory in CSS that lives in `/src`. The workaround here was to use our existing styles that we build in `ProjectComponentsBaseMap.js`